libmscore/measure.cpp

```diff
@@ -49,7 +49,7 @@
 #include "part.h"
 #include "pedal.h"
 #include "pitchspelling.h"
-#include "repeat.h"
+#include "measurerepeat.h"
```

Comment:
I've commented about this before, but it bears "repeating" here, since GitHUb at first declined to show me the diff on this file because it was so big: this was a big diff that wasn't really a functional change at all.  The bulk of it was just renaming of variables.  If there would be any way to add comments - here on GitHub, not in the source - to save reviewers the trouble of figuring out what has really changed and what has not, it would be a big help.

Reply:
Luckily, I have all the original commits preserved in another branch, so this should be easy.

Reply:
Done